code/ui/manager/src/components/preview/utils/stringifyQueryParams.tsx

```diff
@@ -1,4 +1,4 @@
-import qs from 'qs';
+import { stringify } from 'picoquery';
 
 export const stringifyQueryParams = (queryParams: Record<string, string>) =>
-  qs.stringify(queryParams, { addQueryPrefix: true, encode: false }).replace(/^\?/, '&');
```

Comment:
I think we need some unit tests to verify `stringifyQueryParams` doesn't change between the two implementations.